src/Modules/LearningTasks/Tutor.LearningTasks.Tests/Integration/LearningTaskAuthoringTests.cs

```diff
@@ -9,9 +9,9 @@
 
 namespace Tutor.LearningTasks.Tests.Integration;
 
-public class LearningTaskTests : BaseLearningTasksIntegrationTest
+public class LearningTaskAuthoringTests : BaseLearningTasksIntegrationTest
```

Comment:
Add folders to integration package. See `KC` module for examples. Then revert these renamings.